api/api/src/main/scala/uk/ac/wellcome/platform/api/models/WorkQuery.scala

```diff
@@ -0,0 +1,72 @@
+package uk.ac.wellcome.platform.api.models
+
+import com.sksamuel.elastic4s.http.ElasticDsl._
+import com.sksamuel.elastic4s.searches.queries.{Query, SimpleStringQuery}
+import com.sksamuel.elastic4s.searches.queries.matches.{
+  MultiMatchQuery,
+  MultiMatchQueryBuilderType
+}
+
+sealed trait WorkQuery {
+  val queryString: String
+  def query(): Query
+}
+
+object WorkQuery {
```

Comment:
Are these queries written down anywhere? Is there a ticket or doc or something I can look at?

Reply:
I don't think so, they are a work in progress.